.github/workflows/check-build-test.yml

```diff
@@ -40,6 +41,14 @@ jobs:
       - name: Test directory structure
         run: util/test_directory_structure.sh
 
+  docs:
+    name: "Check for Doxygen problems"
```

Comment:
`"Check doxygen"`, otherwise the name is cut off in the Github Actions tab for me :frowning_face: